### PR TITLE
Update to spack-stack-v1/skylab-3.0.0

### LINKED ISSUE #206: Update build for new spack stack version
Started in `feature/updateBuild` for spack-stack-v1/skylab-3.0.0

Base branch: `feature/1_pythonConfigParse`

### PR DESCRIPTION
**Pull request by jjguerrette**
### Description
Updates the build to use `spack-stack-v1/skylab-3.0.0` modules on Cheyenne.  Currently this only works for GNU, but intel builds are broken with the previous version of spack-stack too.  So this does not remove any functionality.

### Issue closed

Closes #206 

### Tests completed
 - [x] 3dvar_OIE120km_WarmStart
 - [x] 3denvar_OIE120km_WarmStart
 - [x] 3dvar_OIE120km_ColdStart
 - [x] 3dvar_O30kmIE60km_ColdStart
 - [x] 3denvar_O30kmIE60km_WarmStart
 - [x] eda_OIE120km_WarmStart

### CODE REVIEW

File: config/mpas/forecast/stream_list.atmosphere.diagnostics [hunk 1]
```diff
@@ -13,48 +13,64 @@ q2
 t2m
 th2m
 mslp
+relhum_50hPa
```
Comment (jjguerrette) [inline config/mpas/forecast/stream_list.atmosphere.diagnostics:16]: Note that the latest build includes 50/100 hPa diagnostics.

Comment (ibanos90) [reply config/mpas/forecast/stream_list.atmosphere.diagnostics:16]: Thanks for this note @jjguerrette

**Comment by ibanos90** [comment]
Hi @jjguerrette, thanks for this PR. A quick question, the `CMakeLists.txt` used for this new build is the same included in the the `build` folder, there was no need to update the tagged versions, correct?

**Comment by jjguerrette** [comment]
> A quick question, the `CMakeLists.txt` used for this new build is the same included in the the `build` folder, there was no need to update the tagged versions, correct?

Nice catch.  `build/CMakeLists.txt` is only updated when we create a new tagged release in the `MPAS-Workflow` repo.  The group has not established how often or under what criteria that tag creation will be performed.  That's open for discussion.  Either way, that updating of `build/CMakeLists.txt` would be done in a standalone PR.

**Comment by ibanos90** [comment]
> > A quick question, the `CMakeLists.txt` used for this new build is the same included in the the `build` folder, there was no need to update the tagged versions, correct?
> 
> Nice catch. `build/CMakeLists.txt` is only updated when we create a new tagged release in the `MPAS-Workflow` repo. The group has not established how often or under what criteria that tag creation will be performed. That's open for discussion. Either way, that updating of `build/CMakeLists.txt` would be done in a standalone PR.

Ok, that's good to know. Thanks!

**Review by ibanos90 — APPROVED** [review]
All tests completed successfully.

**Review by junmeiban — APPROVED** [review]
Tested scenarios/3denvar_OIE120km_WarmStart. Thank you so much @jjguerrette

**Review by liujake — APPROVED** [review]
You may need to merge this one before merging 202.

**Comment by jjguerrette** [comment]
I can create a branch with these changes based off of `develop` instead of `feature/1_pythonConfigParse` if that is preferred.  Is that what you are suggesting @liujake?

**Comment by liujake** [comment]
I am just not sure you will need to merge this into #202 's feature branch or will change the base branch to 'develop' after #202 is merged. Whatever order of merges is Ok with me.

**Pull request merged**